PFW-1497 Change Eject filament -> Eject from MMU

DRracer opened the pull request:
TODO add missing translations for:
- [x] DE
- [x] ES
- [x] HR
- [x] HU
- [x] NL
- [x] NO
- [x] RO
- [x] SK
- [x] SV

PFW-1497

Commit:
Change Eject filament -> Eject from MMU

PFW-1497

Commit:
ES translation

Commit:
Add missing translations + shorten to 16 chars

3d-gussner commented:
@stelgenhof @leptun @Hauzman @AttilaSVK @Prime1910 @ingbrzy @shatter136 @Painkiller56 @OS-kar Please review the auto translated messages if they are correct in your language.
:+1:  correct
:-1:  needs update and please add a comment what needs to be changed

3d-gussner reviewed (approved):
All looks correct and do not exceed the 16 char limit.
Hope the community translators will confirm the changes.

AttilaSVK commented:
👎 the correct translation should be "Kiadás az MMU-ból", which is too long, so my suggestion is "Kiadás az MMUból" or "Kiadas az MMUbol" without accents.

Commit:
Update HU translation

Hauzman commented:
Ro translation looks good to me

stelgenhof commented:
The Dutch translation looks good to me.

OS-kar commented on lang/po/Firmware_no.po:
```diff
@@ -501,11 +501,11 @@ msgstr "ERR Vent på Bruker"
-msgid "Eject filament"
-msgstr "Mat ut filament"
+msgid "Eject from MMU"
+msgstr "Utkast fra MMU"
```
```suggestion
msgstr "Løs ut fra MMU"
```

DRracer replied:
Done :+1:

Commit:
Update NO translation

Prime1910 commented:
Croatian looks good :)

ingbrzy commented:
For SK a small typo corrected

https://github.com/DRracer/Prusa-Firmware/pull/79/commits/0d3c69c41e56a88b355f64af334a494d1ade9eca

3d-gussner committed:
Update SK po file

Thanks to @ingbrzy see https://github.com/prusa3d/Prusa-Firmware/pull/4023#issuecomment-1436084153
